data/json/monsters/mutant_mammal.json

```diff
@@ -364,5 +445,160 @@
+    },
+    "flags": [ "ANIMAL", "DOGFOOD", "CANPLAY", "HEARS", "KEENNOSE", "PATH_AVOID_DANGER_1", "SEES", "SMELLS", "WARM" ]
+  },
+  {
+    "id": "mon_bigdog_gottweiler",
+    "type": "MONSTER",
+    "name": { "str": "Gottweiler" },
+    "description": "This Rottweiler has grown to the proportions of a cow.  Its short fur is riddled with scars and the hide underneath has grown thick and leathery.",
+    "//": "For mutated dogs, trying to exaggerate what they were good at",
+    "default_faction": "dog",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "450000 ml",
+    "weight": "450 kg",
+    "hp": 330,
+    "speed": 130,
+    "attack_cost": 110,
+    "material": [ "flesh" ],
+    "symbol": "D",
+    "color": "light_gray",
+    "aggression": -10,
+    "morale": 40,
+    "aggro_character": false,
+    "melee_skill": 6,
+    "melee_dice": 4,
+    "melee_dice_sides": 4,
+    "melee_damage": [ { "damage_type": "cut", "amount": 8 } ],
+    "armor_elec": 1,
+    "dodge": 2,
+    "armor_bash": 8,
+    "armor_cut": 15,
+    "armor_bullet": 5,
+    "regenerates": 3,
+    "anger_triggers": [ "HURT", "PLAYER_NEAR_BABY", "FRIEND_ATTACKED", "PLAYER_CLOSE" ],
+    "harvest": "mutant_mammal_large_leather",
+    "dissect": "dissect_lupine_sample_single",
+    "families": [ "prof_intro_biology" ],
+    "reproduction": { "baby_monster": "mon_bigdog_gottweiler_pup", "baby_count": 4, "baby_timer": 360 },
+    "//2": "Given the way bigger pups, decreased their number and increased gestation time",
+    "biosignature": { "biosig_item": "feces_dog", "biosig_timer": 2 },
+    "zombify_into": "mon_dog_zombie_brute",
+    "flags": [ "SEES", "HEARS", "SMELLS", "ANIMAL", "PET_MOUNTABLE", "CANPLAY", "PATH_AVOID_DANGER_1", "WARM" ]
+  },
+  {
+    "id": "mon_bigdog_gottweiler_pup",
+    "type": "MONSTER",
+    "name": { "str": "Gottweiler puppy", "str_pl": "Gottweiler puppies" },
+    "description": "An adorable, defenseless Gottweiler puppy.  Much safer to tame than an adult dog, in spite of its already impressive size.",
+    "default_faction": "dog",
+    "bodytype": "dog",
+    "categories": [ "WILDLIFE" ],
+    "species": [ "MAMMAL" ],
+    "volume": "7500 ml",
+    "weight": "10 kg",
+    "hp": 33,
+    "speed": 105,
+    "material": [ "flesh" ],
+    "symbol": "d",
+    "color": "light_gray",
+    "aggression": -11,
+    "morale": 4,
+    "melee_skill": 2,
+    "melee_dice": 1,
+    "melee_dice_sides": 1,
+    "melee_damage": [ { "damage_type": "cut", "amount": 3 } ],
+    "dodge": 1,
+    "regenerates": 3,
+    "vision_night": 5,
+    "harvest": "dog_small_with_skull",
+    "path_settings": { "max_dist": 10 },
+    "upgrades": { "age_grow": 42, "into": "mon_bigdog_gottweiler" },
+    "zombify_into": "mon_dog_zombie_rot",
+    "biosignature": { "biosig_item": "feces_dog", "biosig_timer": 6 },
+    "fear_triggers": [ "FRIEND_ATTACKED", "FRIEND_DIED", "HURT" ],
+    "special_attacks": [ [ "EAT_FOOD", 100 ] ],
```

Comment:
Either all dogs eat food or none do, one puppy doing so is very inconsistent.

Reply:
The idea here was that the puppies would be tamable but the adults would not due to them being literally the size where they'd instead be able to crunch on you as the player. Both mutant puppies should probably do that, though.

Reply:
I mean specifically the `EAT_FOOD` special.